ui/raidboss/raidboss_config.js

```diff
@@ -250,6 +251,27 @@ function setOptionsFromOutputValue(options, value) {
   }
 }
 
+// Helper for doing nothing during trigger eval, but still recording any
+// calls to `output.responseOutputStrings = x;` via callback.
+class DoNothingFuncProxy {
+  constructor(outputStringsCallback) {
+    return new Proxy(this, {
+      set(target, property, value) {
+        if (property === 'responseOutputStrings') {
+          outputStringsCallback(value);
+          return true;
+        }
+
+        throw new Error('Invalid set of property ${property}');
```

Comment:
In both places, this is explicitly `throw`ing; nothing wrong with it, just that I don't frequently see it in cactbot code and want to make sure it's intentional.

Reply:
That's fair.  We could return '???' or even '?? ERROR ??' or something more obvious, so it appears on screen and also put something in the log.  Does that sound better?

Reply:
What happens if you have a borked trigger? I think it just doesn't play, right? If so, we could arguably be better, but I think logging + maybe something that's easy and doesn't need a translation, like you said with `?? ERROR ??`.

Reply:
Ah, ok ok.  These exceptions are only for setting values.  Getting a bogus value or param already returns `???`.  I still think it's probably better to be kind to user triggers and just print an error.